app/scripts/lib/pending-tx-tracker.js

```diff
@@ -99,8 +100,8 @@ module.exports = class PendingTransactionTracker extends EventEmitter {
   }
 
   async _resubmitTx (txMeta) {
-    if (txMeta.retryCount > this.retryLimit) {
-      const err = new Error(`Gave up submitting after ${this.retryLimit} blocks un-mined.`)
+    if (Date.now() > txMeta.time + this.retryTimePeriod) {
+      const err = new Error(`Gave up submitting after ${this.retryTimePeriod / 3.6e+6} hours.`)
```

Comment:
Might be nice to round this down:
`const hours = (this.retryTimePeriod / 3.6e+6).toFixed(1)`